src/core/tools/common/raw_message_parser.cpp

```diff
@@ -43,8 +43,9 @@
 
 namespace dsn {
 
-//static
-std::atomic_bool raw_message_parser::s_handler_hooked(false);
+DEFINE_TASK_CODE_RPC(RPC_CALL_DISCONNECT, TASK_PRIORITY_COMMON, THREAD_POOL_DEFAULT)
```

Comment:
maybe change to "RPC_CALL_RAW_DISCONNECT" and "RPC_CALL_RAW_MESSAGE" ? all is prefixed by "RPC_CALL_RAW_" to avoid conflicting with others.